cmd/version/version_test.go

```diff
@@ -26,12 +26,12 @@ func TestVersionWorksWithoutAccessibleConfigFile(t *testing.T) {
 	}
 	// re-wire
 	oldOsStdout := os.Stdout
-	oldConfigPath := config.ConfigPath
-	config.ConfigPath = path
+	oldConfigPath := config.GetConfigPath()
+	_ = config.SetConfigPath(path)
```

Comment:
Anywhere you see this in a test

    _ = config.SetConfigPath(path)

It should be

    require.NoError(t, config.SetConfigPath(path))

There are rather a lot of these in this, and I'll just comment on this one!